.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.16 # please change also in `release.yml` and `go.mod`
+          go-version: 1.16 # please change also in `release.yml`,`update-generated-docs.yml` and `go.mod`
```

Comment:
BTW keeping this as a Github Secret would be an easy way to share a single version between all the files.

Reply:
https://github.com/env0/terraform-provider-env0/pull/228/commits/21398293c0c48588bcefd4e55317fb4bd05a5bc9